Automattic/wp-calypso
# Add quick filters for domain upsell experiment

#### Proposed Changes

Desktop

https://user-images.githubusercontent.com/6586048/216567392-5782ba59-9b1f-49b4-986f-a2909532d000.mp4

Smaller sizes

https://user-images.githubusercontent.com/6586048/216568484-1b65d1a0-dec8-4d3a-94b8-54acb2abfd2b.mp4

* Added quick filters to the bottom of the search bar

#### Testing Instructions



* Go to `/domains/add/:site?domainAndPlanPackage=true`
* Click around the filters to make sure nothing breaks
* Check for all screen sizes, should be able to swipe on mobile, have More dropdown, etc.

#### Pre-merge Checklist





Fixes #72894
